articles/libraries/auth0js/index.md

```diff
@@ -39,591 +39,178 @@ Install via [bower](http://bower.io):
 ```sh
 bower install auth0.js
 ```
+```html
+<script src="bower_components/auth0.js/build/auth0.min.js"></script>
+```
 
 Include via our CDN:
 
 ```html
-<script src="${auth0js_url}"></script>
+<script src="${auth0js_8_url}"></script>
 ```
 
-If you are using [browserify](http://browserify.org/), you will want to install with `npm i auth0-js --production --save`.
+>Note that for production use, the latest patch release (for example, 8.0.0) is recommended, rather than the latest minor release indicated above.
 
-### Initialize
+If you are using [browserify](http://browserify.org/), you will want to install with `npm i auth0-js --production --save`.
 
-> Note: The following examples use jQuery, but auth0.js is not tied to jQuery and any library can be used with it.
+### Initialization
 
-Construct a new instance of the Auth0 client as follows:
+Initialize a new instance of the Auth0 client as follows:
 
 ```html
 <script src="${auth0js_url}"></script>
 <script type="text/javascript">
-  var auth0 = new Auth0({
+  var auth0 = new auth0.WebAuth({
```

Comment:
can we use a different variable name? it will collition with the auth0 namespace.

Lets use something like webAuth, auth0Client or so.